src/pyproject_indirect_import_detector/detector.py

```diff
@@ -0,0 +1,99 @@
+import logging as _logging
+
+# noqa idiom
+if True:
+    logger = _logging.getLogger(__name__)
+    handler = _logging.StreamHandler()
+    _level = _logging.WARNING
+    handler.setLevel(_level)
+    logger.setLevel(_level)
+    logger.addHandler(handler)
+    logger.propagate = False
```

Comment:
Your knowledge is too old. See https://github.com/Idein/actfw-gstreamer/pull/6/files .

Reply:
9c1d354